Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15.5-alpine AS build
+FROM golang:1.17.8-alpine AS build
 
```

Comment:
This should be 

    FROM golang:1.18-alpine3.15 AS build

we should have always been specifying the alpine version so musl versions would match.